### ISSUE TITLE
Repl.it Support

### ISSUE THREAD

**Issue by MAR0002** [NightZan999/TheImperialGod#37]
Hey is it possible if you can add repl.it support i get this if i try to do it thanks : )
![image](https://user-images.githubusercontent.com/45024391/113500299-2c720100-9560-11eb-9af5-a1531ac3a353.png)

**Comment by NightZan999** [comment]
Hmmm, why we haven't added repl.it support is for multiple reasons that I don't think we can control:
* Repl.it servers are small and **not to be tampered with huge bots like this one**
* It doesn't install latest dpy version (can be helped with `pip install discord.py --upgrade`)
* Your bot will **instantly get ratelimited in a huge server where lots of tickets are opened**

As for the issue, could you please describe which version of the bot you're running?
The latest from main? Make sure you clone all the files. 

Basically you need to clone the cogs folder **also. This error makes me think there is no _cogs folder._**

**Comment by Calciumcarbonate321** [comment]
Make sure both your cogs folder and bot.py folder are in the same directory, then it will work properly.
